docs/guide.mdx

```diff
@@ -61,10 +61,12 @@ If an access token isn't provided, the following rate limits apply:
 The Wave does not support the use of sha256 digest in the image name, e.g. `ubuntu@sha256:3235...ce8f`, when using
 the augmentation process to extend container images.
 
-In order to reference a container via sha256 digest in the image name with Wave you will need to *freeze* image mode
-that will force the creation of a new container image using the container you have specified as base image.
+Whilst using a digest directly isn't possible, Wave can generate image names which include a sha256 digest
+of the `Dockerfile` used to build the image within the tag.
+In order to reference a container via this sha256 digest tag, you will need to use the _freeze_ image mode.
+This will force the creation of a new container image, using the container you have specified as base image.
```

Comment:
Now I'm understanding better what you but it's not correct and a complete different thing what I'm trying to say above. 

The checksum created by using freeze is NOT a sha256 digest (it uses a complete different algo). 

The above above is explaining how you *can* use a container sha256 digest name reference with Wave, not how to produce a container having a digest in the name

Reply:
> The above above is explaining how you can use a container sha256 digest name reference with Wave

Ok great, this PR was part docs and partly intended to try to clear this up so that I understand 😅 

Can you give an example Wave URI that uses a container sha256 digest, without placeholders please? As I don't follow how it works from the docs that were here already.

Reply:
```
 docker pull ubuntu@sha256:3f85b7caad41a95462cf5b787d8a04604c8262cdcdf9a472b8c52ef83375fe15
```

Reply:
Ok but that's skipping wave completely right? I thought you said this morning that you didn't want to do that?

That's what the `nextflow inspect` stuff I suggested before was about, to show how to get these URIs..

Reply:
Not sure to follow. Do you mean how to use digest with wave? it's enough to enable freeze 

```
wave -i ubuntu@sha256:3f85b7caad41a95462cf5b787d8a04604c8262cdcdf9a472b8c52ef83375fe15 --freeze --build-repo docker.io/pditommaso/wave-test
``` 

Not getting how `inspect` is related here

Reply:
Ok that's requesting an image with a digest from Wave. There's no `docker pull` with a digest there?

Reply:
There's a digests for any other container image, but it has nothing do to wit the original one and it's not returned  by `inspect`, so I see now point to mention here